src/App.vue

```diff
@@ -62,12 +82,43 @@ export default {
       Vue.set(this.todos, index, {
         ...this.todos[index],
         done: true,
+        inEditState: false,
       })
     },
 
     handleDelete(id) {
```

Comment:
```suggestion
    onDelete(id) {
```

Please do this in all other places.